corputils/views.py

```diff
@@ -56,26 +69,31 @@ def corp_member_view(request, corpid = None):
     characters_with_api = {}
     characters_without_api = {}
 
+    ncharacters_with_api = 0
```

Comment:
if its not a typo as now i see below it wasn't if it could be renamed to something a little more descriptive so when someone is reading it there not wondering what it's for without having to read through the whole thing.